packages/account/src/Sections/Security/ConnectedApps/connected-apps-empty.tsx

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 import { Text } from '@deriv/components';
-import { observer, useStore } from '@deriv/stores';
+import { observer } from '@deriv/stores';
 import { Localize } from '@deriv/translations';
+import { useDevice } from '@deriv-com/ui';
 import ConnectedAppsInfoBullets from './connected-apps-info-bullets';
 
 const ConnectedAppsEmpty = observer(() => {
```

Comment:
We can remove observer from here as well.

Reply:
done